### PR TITLE
Use 2016.04.00 CDN

### PR DESCRIPTION
**Pull request by randomorder**

### CODE REVIEW

File: web/client/index.html [hunk 1]
```diff
@@ -7,16 +7,16 @@
         <title>MapStore 2 HomePage</title>
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway" type='text/css'>
         <link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css" />
-        <link rel="stylesheet" href="http://rawgit.com/Leaflet/Leaflet.draw/v0.2.4/dist/leaflet.draw.css" />
+        <link rel="stylesheet" href="http://cdn.mapstore2.geo-solutions.it/2016.04.00/leaflet/leaflet.draw.css" />
```
Comment (mbarto) [inline web/client/index.html]: Are you sure we also copy the leaflet css on cdn?

File: web/client/index.html [hunk 2]
```diff
@@ -7,16 +7,16 @@
...
-        <link rel="stylesheet" href="https://rawgit.com/geosolutions-it/MapStore2-theme/master/theme/default/css/ms2-theme.css">
-        <link rel="stylesheet" href="https://rawgit.com/geosolutions-it/MapStore2-theme/master/theme/default/css/font.css">
+        <link rel="stylesheet" href="http://cdn.mapstore2.geo-solutions.it/2016.04.00/theme/default/css/css.css">
+        <link rel="stylesheet" href="http://cdn.mapstore2.geo-solutions.it/2016.04.00/theme/default/css/ms2-theme.css">
+        <link rel="stylesheet" href="http://cdn.mapstore2.geo-solutions.it/2016.04.00/theme/default/css/font.css">
         <script src="http://maps.google.com/maps/api/js?v=3"></script>
         <script src="http://cdnjs.cloudflare.com/ajax/libs/proj4js/2.3.10/proj4.js"></script>
         <script src="http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"></script>
-        <script src="http://rawgit.com/Leaflet/Leaflet.draw/v0.2.4/dist/leaflet.draw-src.js"></script>
+        <script src="http://cdn.mapstore2.geo-solutions.it/2016.04.00/leaflet/leaflet.draw-src.js"></script>
```
Comment (mbarto) [inline web/client/index.html]: And also this one?

File: web/client/index.html (hunk 1 above)
Comment (randomorder) [inline web/client/index.html]: File exists on CDN

File: web/client/index.html (hunk 2 above)
Comment (randomorder) [inline web/client/index.html]: File exists on CDN

**Pull request merged**